unittests/test-contracts/kv_test/kv_test.abi

```diff
@@ -294,5 +256,6 @@
     ],
     "tables": [],
     "ricardian_clauses": [],
-    "variants": []
+    "variants": [],
+    "action_results": []
```

Comment:
Is "action_results" from some other new feature?

Reply:
Yes, these contracts just had not been rebuilt since that was added.